db/queries/application/queries.py

```diff
@@ -227,14 +227,22 @@ def process_files(application: Applications, all_files: list[FileData]) -> Appli
     return application
 
 
-def update_project_name(form_name, question_json, application) -> None:
-    if form_name.startswith("project-information") or form_name.startswith(
-        "gwybodaeth-am-y-prosiect"
-    ):
+def update_project_name(
+    form_name, question_json, application, project_name_field_id=None
+) -> None:
+    forms_containing_project_name = [
+        "project-information",
+        "gwybodaeth-am-y-prosiect",
+        "name-your-application",
+    ]
+    if any(form_name.startswith(item) for item in forms_containing_project_name):
```

Comment:
Do we want to store the form names in the code? We've moved so much other config to the DB this seems to go against that. As we've got the field key, could we do something like 
`project_name = filter(lambda x:x["question"]["fields"]["key"]== project_name_field_id, question_json)`
I'm sure that would need tweaking but then we are only using info from the DB, not hard coding anything.

Reply:
yes I agree, as far as I know this form is a temporary solution (kind of a patch by design) to this project name problem. I did not want to change our schema until there was a more clear approach (e.g the project-name/name-your-application form could be a universally consistent and we would need this list). What do you both think?

Reply:
ignore my last comment, i see we have the field_id in the cof config : ) ill try to hook into this